OxfordBodleian/Juel-Jensen/BDLaethg39.xml

```diff
@@ -53,25 +210,184 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                    elements of numerals and punctuation marks. Traces of rubrication are also visible on the verso.</seg>
                             </handNote>
                         </handDesc>
                         
-                       <!-- <decoDesc>
-                         
+                     <!--   <decoDesc>
+                            <decoNote xml:id="d1" type="miniature" corresp="#pic1">
+                                <locus target="#1r"/>
+                                <desc></desc>
+                                <dimensions unit="mm">
+                                    <height>117</height>
+                                    <width>89</width>
+                                </dimensions>
+                            </decoNote>
+                            
+                            <decoNote xml:id="d2" type="miniature" corresp="#pic2">
+                                <locus target="#1r"/>
+                                <desc></desc>
+                                <dimensions unit="mm">
+                                    <height>127</height>
+                                    <width>90</width>
+                                </dimensions>
+                            </decoNote>
+                            
+                            <decoNote xml:id="d3" type="miniature" corresp="#pic3">
+                                <locus target="#1r"/>
+                                <desc></desc>
+                                <dimensions unit="mm">
+                                    <height>127</height>
+                                    <width>82</width>
+                                </dimensions>
+                            </decoNote>
+                        
+                            <decoNote xml:id="d4" type="miniature" corresp="#pic4">
+                                <locus target="#1v"/>
+                                <desc></desc>
+                                <dimensions unit="mm">
+                                    <height>102</height>
+                                    <width>87</width>
+                                </dimensions>
+                            </decoNote>
+                        
+                            <decoNote xml:id="d5" type="miniature" corresp="#pic5">
+                                <locus target="#1v"/>
+                                <desc></desc>
+                                <dimensions unit="mm">
+                                    <height>85</height>
+                                    <width>87</width>
+                                </dimensions>
+                            </decoNote>
                         </decoDesc>-->
                         
+                        <additions>
+                            <list>
+                                <item xml:id="e1">
+                                    <desc>The verso of the scroll seems to mirror its recto: It contains 
```

Comment:
perhaps" contained" since neither the miniatures nor the text are now clearly visible?

Reply:
Sorry, but I am confused by this: is this not simply the ink that has penetrated into the parchment due to e.g. dampness?

Reply:
You are completely right, thank you!
I included the information in condition (which I had previously forgotten...).